react/features/chat/reducer.js

```diff
@@ -22,14 +22,22 @@ ReducerRegistry.register('features/chat', (state = DEFAULT_STATE, action) => {
             timestamp: action.timestamp
         };
 
+        // React native, unlike web, needs a reverse sorted message list.
```

Comment:
I think we also need to change how lastReadMessage is computed, in the TOGGLE_CHAT action

Reply:
True, as well as the ```getUnreadCount``` function. Thanks!